dev/ts/shared/main.ts

```diff
@@ -301,7 +303,9 @@ function configurePoolsFilteringFromParameters(parameters: URLSearchParams): Mod
 
 export function startBpmnVisualization(config: BpmnVisualizationDemoConfiguration): void {
   const log = logStartup;
-  log(`Initializing BpmnVisualization with container '${config.globalOptions.container}'...`);
+  const container = config.globalOptions.container;
+  const containerId = typeof container === 'string' ? container : container.id;
+  log(`Initializing BpmnVisualization with container '${containerId}'...`);
```

Comment:
**nitpick:** previously, when the container was an HTML element, it was fully logged on purpose.
The container id is not mandatory, so in this case undefined would be logged.
So I suggest to restore the previous implementation.

Reply:
With the previous implementation ``log(`Initializing BpmnVisualization with container '${config.globalOptions.container}'...`);`` and the `recommended-type-checked` ESLint rule, we have the errors:
```
ESLint: 'config.globalOptions.container' may evaluate to '[object Object]' when stringified.(@typescript-eslint/no-base-to-string)
ESLint: Invalid type "string | HTMLElement" of template literal expression.(@typescript-eslint/restrict-template-expressions)
```